atima/README.md

```diff
@@ -0,0 +1,30 @@
+# AtimaWrapper for R3BRoot
+
+You can get energyloss etc. from Atima inside R3BRoot with only a few lines of code.
+The following line, e.g. will give you the Energyloss of a proton beam @ 400 MeV in a 50mm liquid hydrogen target:
+
+```c++
+    R3BAtima::Calculate_mm(1, 1, 400, R3BAtimaTargetMaterial::LH2, 50).ELoss_AMeV;
+``` 
+
+You can define Materials using R3BAtimaTargetMaterial and R3BAtimaMaterialCompound or use predefined Materials inside R3BAtimaTargetMaterial.
+Please add commonly used materials which are not defined there yet!
+
+Since the computation takes a small amount of time, you can store the result in a chosen range in a so called map in order to increase the speed in frequent computations.
```

Comment:
```suggestion
Since the computation takes a small amount of time, you can cache the result in a chosen range in order to increase the speed in frequent computations.
```